<launch>

  <arg name="cam_1" default="camera_1" />
  <!-- <arg name="cam_2" default="camera_2" />
  <arg name="cam_3" default="camera_3" /> -->

   <node pkg="gige_cam_driver" type="camera_node.py" name="$(arg cam_1)" output="screen">
      <param name="device_id" value="0"/>
      <param name="camera_manager" value="$(arg cam_1)"/>
      <param name="calibration_file" value="$(find gige_cam_driver)/config/camera_info_cam1.yaml"/>
   </node>


  <node pkg="camera_calibration" name="camera_1_calibration" type="cameracalibrator.py" output="screen">
    <param name="size" value="6x5"/>
    <param name="square" value="0.025"/>
    <param name="k-coefficients" value="2"/>
    <param name="fix-principal-point" value="i"/>
    <param name="fix-aspect-ratio" value="true"/>
    <remap from="image" to="/camera_1/image_raw"/>
    <remap from="camera" to="/camera_1"/>
  </node>
</launch>

<!-- rosbag record --duration=20 --output-name=/home/agcam/catkin_ws/src/bagfiles/camera1.bag /camera_1/camera_info /camera_1/image_raw -->